microfrontends/app-builder-menu/src/components/MenuUI.tsx

```diff
@@ -450,9 +461,17 @@ export function MenuUI(props: Props): JSX.Element {
                     })}
                   </SecondaryMenuItem>
                 );
-              })}
-            </ListGroupItem>
-          )}
+              })
+            ) : (
+              <StyledPlaceholder
+                href="https://hub.entando.com/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Install your first Entando Platform Capability
```

Comment:
we should translate also this content, here are the translations:

**it**: Installa la tua prima Entando Platform Capability
**pt**: Instale sua primeira Entando Platform Capability

please use always content.ts file when you insert new text.